python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -161,6 +161,8 @@ def run_model(repo_id, test_api, in_out_pairs, local_model_hub=None, warm_up=1,
         result = run_speculative_gpu(repo_id, local_model_hub, in_out_pairs, warm_up, num_trials, num_beams, batch_size)
     elif test_api == 'pipeline_parallel_gpu':
         result = run_pipeline_parallel_gpu(repo_id, local_model_hub, in_out_pairs, warm_up, num_trials, num_beams, low_bit, batch_size, cpu_embedding, fp16=use_fp16_torch_dtype)
+    elif test_api == 'transformer_win_npu_int4':
```

Comment:
Should we call it `'transformers_int4_npu_win'`? To make the `test_api`'s name be consistent with others